docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -76,6 +76,10 @@ spec:
         count: 2
 ```
 
+#### Combining Policies
+
+The authorizer applies require policies within a role together with an OR operator and the sets from each role with an AND operator. In practice, this means that every role with at least one require policy requires one of its policies to be met before a session can be started.
```

Comment:
- This is the first time we mention the authorizer. Is this the same as the Auth Service? If so, we should say "The Auth Service".
- Is there a term defined earlier in the guide we can use instead of "the sets from each role"? I'm not confident that I understand what this refers to.

Reply:
1. The authorizer usually a common term for some control logic at any point in a stack that decides whether a user has access or not. This isn't a term frequently used in documentation I don't think but I don't have a better name for it since this code can be run at many different places.

2. Reworded, is it clearer now?